application/src/Permissions/Assertion/OwnsSiteAssertion.php

```diff
@@ -0,0 +1,20 @@
+<?php
+namespace Omeka\Permissions\Assertion;
+
+use Omeka\Entity\SitePage;
+use Laminas\Permissions\Acl\Acl;
+use Laminas\Permissions\Acl\Assertion\AssertionInterface;
+use Laminas\Permissions\Acl\Resource\ResourceInterface;
+use Laminas\Permissions\Acl\Role\RoleInterface;
+
+class OwnsSiteAssertion implements AssertionInterface
```

Comment:
Was there an issue with the previous way this was working with OwnsEntityAssertion in your first version of this PR? Since SitePage has getOwner implemented (as a passthrough to the containing Site's getOwner) it seems to me that it should work fine without needing a one-off assertion like this, and I think it's preferable to have it work that way.

Reply:
Too much in advance.
I removed it.